### PR TITLE
Add homepage card for iframe plugin

### PR DESCRIPTION
**Pull request by nicolasarnold12321**
[ch8171]

#### :heavy_check_mark: Checklist

- [ ] Added tests for new functionality and regression tests for bug fixes
- [ ] Added changeset (run `yarn changeset` in the root)
- [ ] Screenshots of before and after attached (for UI changes)
- [ ] Added or updated documentation (if applicable)

### CODE REVIEW

File: plugins/frontend/backstage-plugin-iframe/src/components/HomePageCard.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    return (
+      <ErrorComponent {...{errorMessage}}/>
+    )
+  }
+
+  if(errorMessage !== ""){
+    return (
+      <ErrorComponent {...{errorMessage}}/>
+    )
+  }
```
Comment (kissmikijr) [inline plugins/frontend/backstage-plugin-iframe/src/components/HomePageCard.tsx:46]: Some suplicaton here, what did you want to check in the second if ?

File: plugins/frontend/backstage-plugin-iframe/src/plugin.ts [hunk 2]
```diff
@@ -54,3 +55,18 @@ export const EntityIFrameContent = iframePlugin.provide(
   }),
 );
 
+export const HomePageIFrameCard = homePlugin.provide(
+  createCardExtension<{
+    src: string;
+    title?: string;
```
Comment (kissmikijr) [inline plugins/frontend/backstage-plugin-iframe/src/plugin.ts:61]: You don't need to give the title prop here or do you use it inside the `./components/HomePageCard` ?

Comment (kissmikijr) [reply plugins/frontend/backstage-plugin-iframe/src/plugin.ts:61]: Ah I see, you use it, so take this as I didn't say anything

File: plugins/frontend/backstage-plugin-iframe/src/components/HomePageCard.tsx (hunk 1 above)
Comment (nicolasarnold12321) [reply plugins/frontend/backstage-plugin-iframe/src/components/HomePageCard.tsx:46]: silly me. Thank you

File: plugins/frontend/backstage-plugin-iframe/src/plugin.ts (hunk 2 above)
Comment (nicolasarnold12321) [reply plugins/frontend/backstage-plugin-iframe/src/plugin.ts:61]: I'll remove it!

**Pull request merged**